django/contrib/postgres/fields/array.py

```diff
@@ -16,6 +18,32 @@
 __all__ = ['ArrayField']
 
 
+class ArrayLiteral:
+    """
+    An object which psycopg2 serializes into an array, but is hashable and
+    comparable like a tuple.
+    """
+
+    def __init__(self, value):
+        self.value = tuple(value)
+
+    def __conform__(self, protocol):
+        if protocol is psycopg2.extensions.ISQLQuote:
+            return psycopg2.extensions.adapt(list(self.value))
+        return None
```

Comment:
Not sure that's needed (it's the implicit default) -- anyway, as noted above, it's never executed.

Reply:
( pep8 suggests including an explicit `return None` if the end of the function is reachable. :) )